docs/attribution.md

```diff
@@ -101,3 +101,83 @@ DOC/NOAA/NESDIS/NCEI > National Centers for Environmental Information, NESDIS, N
 ```
 
 More details can be found on the ETOPO1 [about](https://www.ngdc.noaa.gov/mgg/global/global.html) page and [data sources](https://www.ngdc.noaa.gov/mgg/global/etopo1sources.html) page, and [description](https://www.ngdc.noaa.gov/docucomp/page?xml=NOAA/NESDIS/NGDC/MGG/DEM/iso/xml/316.xml&view=getDataView&header=none) page and [disclaimer](https://www.ngdc.noaa.gov/ngdcinfo/privacy.html#disclaimer) page.
+
+### Land Information New Zealand (LINZ)
+
+[Released](https://data.linz.govt.nz/layer/1768-nz-8m-digital-elevation-model-2012/) by [LINZ](https://data.linz.govt.nz/), licensed under [Creative Commons Attribution 3.0 New Zealand](https://data.linz.govt.nz/license/attribution-3-0-new-zealand/).
+
+Attribution statement:
+
+```
+Copyright 2011 Crown copyright (c) Land Information New Zealand and the New Zealand Government. All rights reserved
```

Comment:
I think this (and all the other new ones) need to be `>` quoted instead of ``` quoted so they will wrap in the Markdown generator – now they extend way over to the right margin of the page.

Reply:
Fixed in 06758f8.